modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIFilterRelevantObjectEntryModelListener.java

```diff
@@ -64,6 +64,22 @@ private void _validate(ObjectEntry objectEntry) {
 					"an-api-filter-must-be-related-to-an-api-endpoint");
 			}
 
+			if (Validator.isNull(
```

Comment:
We can try to unify the logic between the 2 model listeners again taking into consideration what he told us in the last PR.